addons/hr_holidays/tests/test_hr_leave_type.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+
+from odoo.addons.hr_holidays.tests.common import TestHrHolidaysBase
+
+
+class TestHrLeaveType(TestHrHolidaysBase):
```

Comment:
I would simplify tests here and just add one simple test that checks that time_type is effectively correctly propagated. As we have in this task currently no functional behavior implemented I would wait for the next tasks to add a behavior and add tests at that time.